core/storage/classroom/gae_models.py

```diff
@@ -135,9 +173,17 @@ def create(
             name=name,
             url_fragment=url_fragment,
             course_details=course_details,
+            teaser_text=teaser_text,
             topic_list_intro=topic_list_intro,
             topic_id_to_prerequisite_topic_ids=(
-                topic_id_to_prerequisite_topic_ids)
+                topic_id_to_prerequisite_topic_ids),
+            is_published=is_published,
+            thumbnail_filename=thumbnail_filename,
+            thumbnail_bg_color=thumbnail_bg_color,
+            thumbnail_size_in_bytes=thumbnail_size_in_bytes,
+            banner_filename=banner_filename,
+            banner_bg_color=banner_bg_color,
+            banner_size_in_bytes=banner_size_in_bytes
```

Comment:
Here as well, and throughout your PR, create a structure with three fields (filename, bg_color, size_in_bytes).

As a rule of thumb, if it's not difficult, try to limit the number of function arguments to 5-6.

Reply:
Done. created ImageDict with filename, bg_color, size_in_bytes and image_data

we will use the Classroom domain object when getting data from frontend and as we will find the image size on the backend hence I have made size_in_bytes optional.